localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+echo ""
+echo "Installing po2json..."
+echo ""
+npm install po2json
+
+echo ""
+echo "Generating .po to .json files..."
+echo ""
+
+pofilenames=$(ls ./translations/planet4-plugin-gutenberg-blocks/languages/*.po)
+for eachpofile in $pofilenames
```

Comment:
I guess just `for pofile` would work as well?

Reply:
Could you please elaborate? 🤔

Reply:
I just meant that the naming probably doesn't have to be prefixed with `each`. We could also include `name` at the end so that it matches up with the list.